apps/cli/src/downloadCodemod.ts

```diff
@@ -86,29 +90,42 @@ export class CodemodDownloader implements CodemodDownloaderBlueprint {
 
 		await mkdir(directoryPath, { recursive: true });
 
-		// download the config
-		const configPath = join(directoryPath, 'config.json');
+		let parsedConfig: unknown;
 
-		const buffer = await this._fileDownloadService.download(
-			`${CODEMOD_REGISTRY_URL}/${hashDigest}/config.json`,
-			configPath,
-		);
+		try {
+			// download the config
+			const configPath = join(directoryPath, 'config.json');
 
-		const parsedConfig = JSON.parse(buffer.toString('utf8'));
+			const buffer = await this._fileDownloadService.download(
+				`${CODEMOD_REGISTRY_URL}/${hashDigest}/config.json`,
+				configPath,
+			);
 
-		const config = S.parseSync(codemodConfigSchema)(parsedConfig);
+			parsedConfig = JSON.parse(buffer.toString('utf8'));
+		} catch (error) {
+			await handleListNamesCommand(this.__printer);
 
-		{
-			const descriptionPath = join(directoryPath, 'description.md');
+			throw new Error(
```

Comment:
should we check if error is actually axios `404` error before throwing this?

Reply:
it's actually 403 because of how AWS handles this, but i could add a check. not sure if it's needed though